galasa-extensions-parent/dev.galasa.auth.couchdb/src/main/java/dev/galasa/auth/couchdb/internal/CouchdbAuthToken.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.auth.couchdb.internal;
+
+import java.time.Instant;
+
+import dev.galasa.framework.spi.auth.IAuthToken;
+import dev.galasa.framework.spi.auth.User;
+
+public class CouchdbAuthToken implements IAuthToken {
+
+    private String _id;
+    private String clientId;
```

Comment:
This is a id of the client which refers to a particular dex refresh token.

Reply:
Renamed to dexClientId since this is a dex-specific ID